_api-reference/profile.md

```diff
@@ -213,6 +221,220 @@ The response contains profiling information:
 ```
 </details>
 
```

Comment:
@kolchfa-aws  Should we move the above "## Concurrent segment search" section to here? It's because "#### Example request" section is not related to concurrent segment search

Reply:
I think it makes sense to leave the section on top because it's an overview. And then there are examples. I'll add a heading to the second example so we differentiate.